app/assets/javascripts/app/discussions/index.html.slim

```diff
@@ -11,9 +11,9 @@ div ui-view="sidebar"
   .TopBarLayout-content
     .Page
       section.DiscussionList layout='column' layout-margin=''
-        md-container.u-upperCase flex='' Open Discussions
-        md-container layout='column' layout-margin='' wait-for='ctrl.pageReady'
+        md-container.DiscussionList-state flex='' Open Discussions
```

Comment:
Do you think this element represents the `state` of the `DiscussionList`? What about line 19?